### PR TITLE
[idea] #295 Added privacy policy file for kodebeagle

### LINKED ISSUE #295: kodebeagle privacy policy for plugin.
Also explain how KodeBeagle plugin works in the plugin.xml.

### PR DESCRIPTION
**Pull request by amit-shar**
fixes #295

### CODE REVIEW

**Comment by amit-shar** [comment]
@prudhvije Please review!

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/LegalNotice.java [hunk 1]
```diff
@@ -56,6 +60,10 @@
     private static final String DIV_STYLE_MARGIN_5PX = "<div style='margin:5px;'>";
     private static final String DIV = "</div>";
     private static final String KODEBEAGLE_NOTICE_FILENAME = "KODEBEAGLE_NOTICE";
+    private static final String KODEBEAGLE_PRIVACY_POLICY =
+            "<html><div style=\"text-align:right\"><a href=\"\" >Privacy Policy</a></div>"
+                    + "<br></html>";
+    private static final String KODEBEAGLE_PRIVACY_POLICY_LINK = "KODEBEAGLE_PrivacyPolicy";
```
Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/LegalNotice.java]: KODEBEAGLE_PRIVACY_POLICY

File: docs/KodeBeaglePrivacyPolicy.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<h1>KodeBeagle Privacy Policy</h1>
+
+<p>KodeBeagle strongly believes that the protection of individuals' privacy is a critical issue when doing business
+    via the Internet. We have created this policy to inform our website visitors and plugin users of the measures we
+    employ to keep private and confidential the information they may choose to share with us.</p>
+
+<p>KodeBeagle does collect some kinds of information from its website/plugin users. By using this website/plugin,
+    you agree to allow us to collect the kinds of information described later on this page, and to use it in the ways
```
Comment (ScrapCodes) [inline docs/KodeBeaglePrivacyPolicy.html]: By using this website/plugin unless opted out of anonymous stats collection,

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/LegalNotice.java [hunk 3]
```diff
@@ -98,11 +107,17 @@ protected final JComponent createCenterPanel() {
...
         messageEditorPane.setBorder(BorderFactory.createLineBorder(Gray._200));
         String text = DIV_STYLE_MARGIN_5PX + getLegalNoticeMessage() + DIV;
-        messageEditorPane.setText(text);
-
+        messageEditorPane.setText(text + "\n" + String.format(PRIVACY_POLICY));
         JPanel legalNoticePanel = new JPanel(LEGAL_NOTICE_LAYOUT);
         legalNoticePanel.add(iconPanel, BorderLayout.WEST);
         legalNoticePanel.add(messageEditorPane, BorderLayout.CENTER);
+
+        messageEditorPane.addMouseListener(new MouseAdapter() {
```
Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/ui/LegalNotice.java]: It seems, you have set the listener on the entire pane ! Not just that link.

**Comment by ScrapCodes** [comment]
merged.

**Pull request closed without merging**